src/Microsoft.AspNetCore.OData/Extensions/ODataServiceCollectionExtensions.cs

```diff
@@ -10,6 +11,10 @@
 using Microsoft.AspNet.OData.Routing;
 using Microsoft.AspNetCore.Mvc.Filters;
 using Microsoft.AspNetCore.Mvc.Infrastructure;
+using Microsoft.AspNetCore.Routing;
+#if NETCOREAPP3_1
+using Microsoft.AspNetCore.Routing.Matching;
```

Comment:
Once we upgrade to higher versions of net core, this would need to be changed right? Is there a conditional compile that specifies greater than?

Reply:
Revert.

---
In reply to: [375983113](https://github.com/OData/WebApi/pull/2035#discussion_r375983113) [](ancestors = 375983113)